understanding/20/keyboard-no-exception.html

```diff
@@ -29,6 +29,28 @@ <h2>Intent of Keyboard (No Exception)</h2>
          Rather, it means that content that uses path-dependent input cannot conform to this
          Success Criterion and therefore cannot meet Guideline 2.1 at Level AAA. 
       </p>
+
+      <div class="note">
+         <p>Platforms and user agents usually have conventions for how web content or applications
+            are controlled with a keyboard interface. As a best practice, content should follow
+            the platform/user agent conventions. However, deviating from these conventions does
+            not fail the normative requirement of this Success Criterion. For instance, buttons
```

Comment:
I think this needs to be softened. I would be very concerned if someone interpreted this to mean that a complete deviation from standard keyboard interaction is never a failure of 2.1.1.
```suggestion
            not necessarily fail the normative requirement of this Success Criterion. For instance, buttons
```

Reply:
when is it a failure, normatively? i don't see anything in the normative text that suggests this